rocm_getpath.cpp

```diff
@@ -39,7 +42,7 @@
 #define TARGET_LIB_INSTALL_DIR TARGET_LIBRARY_INSTALL_DIR
 
 /* Target Library Name Buf Size */
-#define LIBRARY_FILENAME_BUFSZ (PATH_MAX+1)
+#define LIBRARY_FILENAME_BUFSZ 4096
```

Comment:
Internal builds of ROCm ci uses a very long path. Need to check if this size is enough

Reply:
I believe `PATH_MAX` from `<limits.h>` is typically set to `4096` on modern Linux systems: https://stackoverflow.com/a/9449307, https://stackoverflow.com/a/73168481. I guess this could be `4096 + 1` for consistency.

Could also look into other cross-platform solutions as discussed on https://stackoverflow.com/questions/65174086/is-there-any-difference-between-filename-max-and-path-max-in-c

Reply:
Thanks I also verified, size should be fine. though hardcoding is not recommended we can take it

Reply:
This looks better solution that hard coding to 4096.
https://stackoverflow.com/questions/65174086/is-there-any-difference-between-filename-max-and-path-max-in-c